lib/cli-config.js

```diff
@@ -72,7 +76,15 @@ exports.getContent = function(config, directory) {
                 )
             );
         }
+    } else {
+
+        // Try to load it as a node module
+        try {
+            content = require(config);
+        } catch (e) {}
```

Comment:
What will the user see if the resolution fails? Said differently, what does the user see if this function returns `undefined`? I assume it'll bubble to cli.js and then show an error about a missing config?